services/ui-src/src/libs/formLib.tsx

```diff
@@ -45,24 +47,57 @@ type ConfirmSubmitType = {
   confirmSubmitYesButton?: string;
 };
 
+export const DefaultFileTypesInfo = () => (
+  <p>
+    We accept the following file types:{" "}
+    <b>
+      .doc, .docx, .jpg, .odp, .ods, .odt, .png, .pdf, .ppt, .pptx, .rtf, .txt,
+      .xls, .xlsx, and a few others.
+    </b>{" "}
+    See the full list on the{" "}
+    <Link to={ROUTES.FAQ_ACCEPTED_FILE_TYPES} target="_blank">
+      FAQ Page
```

Comment:
I think I wasn't clear on the target="_blank" versus target="New" request - In my opinion, they should all be the same, either all new windows or the singleton window named "new" - right now, there are five links to the FAQ page on a form page, 2 go to "New" and 3 open a new window.

Reply:
None of them open a new window on my end, everything opens a new tab only. If we want to file the change for other links as a tech debt 1-pointer we can, but I fail to see how this relates to the AC of _this_ ticket. [According to the HTML5 spec](https://html.spec.whatwg.org/multipage/document-sequences.html#browsing-context-names), we should be using `_blank`.

Reply:
I have very little opinion on what way they work, I personally would like all FAQ links on a page to work the same - which they do not do in my Chrome browser.  But this is not in the AC and there is certainly room for disagreement, so I will retract my request for this change.

Reply:
I think I'm gonna demo from this PR's environment link, and pose the question so we know from the top what the expected behavior is (or at least convince them to go one way or the other if they don't have an opinion) because they should be uniform, and I think they'll have a preference even if we don't. Like I said, quick change and it'd be good to go, but I'm kinda in a "_always_ have a requirement first" state of mind due to the vague ACs lately.